scalecodec/type_registry/polymesh.json

```diff
@@ -797,14 +828,8 @@
         ["PortfolioCustody", "PortfolioId"],
         ["Custom", "Ticker"],
         ["NoData", "Null"],
-        ["TransferCorporateActionAgent", "Ticker"]
-      ]
-    },
-    "AuthIdentifier": {
-      "type": "struct",
-      "type_mapping": [
-        ["signatory", "Signatory"],
-        ["auth_id", "u64"]
+        ["TransferCorporateActionAgent", "Ticker"],
+        ["BecomeAgent", "(Ticker,AgentGroup"]
```

Comment:
aren't we missing the closing brackets `)` here ?

Reply:
good catch, indeed